### PR TITLE
i18(zh-cn): translate `references/v2/acl.mdx` and `references/v2/js/index.mdx`

### PR DESCRIPTION
**Pull request by GuoJikun**
#### What kind of changes does this PR include?

- Translated content

#### Description

translate contents:

- `references/v2/acl.mdx`
- `references/v2/js/index.mdx`

Fix link jump errors:

- `features/autostart.mdx`
- `features/barcode-scanner.mdx`
- `features/cli.mdx`
- `features/deeping-linking.mdx`
- `features/global-shortcut.mdx`
- `features/logging.mdx`
- `features/window-state.mdx`

### CODE REVIEW

File: src/content/docs/zh-cn/references/v2/acl.mdx
Comment (Embers-of-the-Fire) [inline src/content/docs/zh-cn/references/v2/acl.mdx]:
```suggestion
通常你会将其设置为 `../gen/schemas/desktop-schema.json` 或 `../gen/schemas/mobile-schema.json`，不过你也可以为特定的目标平台定义功能。
```

typo: missing whitespace

File: src/content/docs/zh-cn/references/v2/acl.mdx
Comment (Embers-of-the-Fire) [inline src/content/docs/zh-cn/references/v2/acl.mdx]:
```suggestion
如果你想允许远程 URL 访问 Tauri 命令(这很危险，需要仔细配置)，你可以使用 `context` 属性。
```

typo: missing whitespace

**Comment by GuoJikun** [comment]
@Embers-of-the-Fire All done.

**Comment by Embers-of-the-Fire** [comment]
> @Embers-of-the-Fire All done.

I'm not the manager of this repo, just happened to see this PR. You'd better contact @dklassic

File: src/content/docs/zh-cn/references/v2/acl.mdx
Comment (dklassic) [inline src/content/docs/zh-cn/references/v2/acl.mdx:40]:
```suggestion
默认情况下，该功能适用于所有目标，但您可以选择 Linux、macOS、Windows、iOS 和 Android 目标的一部分。
```

File: src/content/docs/zh-cn/references/v2/acl.mdx
Comment (dklassic) [inline src/content/docs/zh-cn/references/v2/acl.mdx]:
```suggestion
为了缓解这种情况，Tauri 使用访问控制列表（Access Control List；ACL）来限制插件命令的访问。默认情况下，所有插件命令都被阻止，无法访问。若要启用对应用程序所需命令的访问，必须定义需要访问的权限列表。
```
Without mentioning the full name first, the abbreviation is kind of meaningless here.

File: src/content/docs/zh-cn/references/v2/js/index.mdx
Comment (dklassic) [inline src/content/docs/zh-cn/references/v2/js/index.mdx:1]: Not quite sure if we need to initialize a localized page for this hummmmm.

But well, I guess this way when it points to API references it'll be marked with untranslated.

**Review by dklassic — CHANGES_REQUESTED** [review]
Thanks for contributing! Just some extra nitpicks and I think we're good to go.

Some extra weird thing I see is that the sidebar doesn't seem to read the localized name properly here, might need to investigate later once we merged.

截圖 2024-03-24 凌晨12 27 48

Also @Embers-of-the-Fire thanks for the initial review! Really appreciate it :)

File: src/content/docs/zh-cn/references/v2/js/index.mdx
Comment (GuoJikun) [reply src/content/docs/zh-cn/references/v2/js/index.mdx:1]: Switching the language to Chinese and clicking on this page shows that there's no translation. So, I did localization.

**Comment by lorenzolewis** [comment]
@dklassic to localise the sidebar you'll need to do it in the config. I believe someone had already done it for another language there. This is because Starlight doesn't allow us to control how deep an auto-generated sidebar is created and so we had to hand-configure some of it there.

File: src/content/docs/zh-cn/references/v2/js/index.mdx
Comment (lorenzolewis) [reply src/content/docs/zh-cn/references/v2/js/index.mdx:1]: Up to @dklassic but in general we only accepted localised pages that were listed on the i18n dashboard. This is a generated page which means if it's localised you won't be able to tell if it's changed or not (since the English version isn't committed).

Comment (dklassic) [reply src/content/docs/zh-cn/references/v2/js/index.mdx:1]: I guess it’s best to stick to the policy and leave it untranslated for now and best to generate it for all languages if we’re to go this route.

**Comment by Embers-of-the-Fire** [comment]
@dklassic Sidebar settings need to be set in the astro config file. I've discussed this in discord before, and my suggestion at the time was to manually write a script to split the localisation to prevent polluting the config file. Other admins seem to prefer to wait for upstream updates, but I can't recall any changes to the current internationalisation of astro regarding this.

**Comment by GuoJikun** [comment]
I didn't see the i18n configuration in the astro profile.

**Comment by Embers-of-the-Fire** [comment]
@GuoJikun You can find it in the `astro.config.mjs` file. Starting at line 98 when the sidebar is configured, there are already one or two localisation settings in there, but it's basically in a completely untranslated state.

**Comment by GuoJikun** [comment]
> @GuoJikun You can find it in the `astro.config.mjs` file. Starting at line 98 when the sidebar is configured, there are already one or two localisation settings in there, but it's basically in a completely untranslated state.

key translations?

File: src/content/docs/zh-cn/references/v2/js/index.mdx
Comment (GuoJikun) [reply src/content/docs/zh-cn/references/v2/js/index.mdx:1]: I need to delete the `zh-cn/references/v2/js/index.MDX ` ? @dklassic

Comment (dklassic) [reply src/content/docs/zh-cn/references/v2/js/index.mdx:1]: Yes let's remove it for now.

Comment (GuoJikun) [reply src/content/docs/zh-cn/references/v2/js/index.mdx:1]: > Yes let's remove it for now.

Ok, removed.

**Comment by GuoJikun** [comment]
If there are no problems and you have time, can we merge? @dklassic

**Pull request merged**
